src/config/constants.ts

```diff
@@ -184,3 +184,5 @@ export const TRIGONOMETRY_SPECIAL_CASES: SpecialCases = {
 
 export const ROUND_OFF_ERROR_MARGIN = 1.58;
 export const BUTTON_FONT_SIZE = '1.9rem';
+
+export const CALCULATION_TRIGGER = 'calculation-operation';
```

Comment:
I think this can be an enum, or an object. We don't know yet, but we might have many triggers.